### ISSUE TITLE
fails to build on musl 32-bit host

### ISSUE THREAD

**Issue by nekopsykose** [greshake/i3status-rust#1741]
```
error[E0308]: mismatched types
   --> src/blocks/disk_space.rs:117:40
    |
117 |         let total = statvfs.blocks() * statvfs.fragment_size();
    |                                        ^^^^^^^^^^^^^^^^^^^^^^^ expected `u64`, found `u32`
error[E0277]: cannot multiply `u64` by `u32`
   --> src/blocks/disk_space.rs:117:38
    |
117 |         let total = statvfs.blocks() * statvfs.fragment_size();
    |                                      ^ no implementation for `u64 * u32`
    |
    = help: the trait `Mul<u32>` is not implemented for `u64`
    = help: the following other types implement trait `Mul<Rhs>`:
              <&'a u64 as Mul<u64>>
              <&u64 as Mul<&u64>>
              <u64 as Mul<&u64>>
              <u64 as Mul>
error[E0308]: mismatched types
   --> src/blocks/disk_space.rs:118:65
    |
118 |         let used = (statvfs.blocks() - statvfs.blocks_free()) * statvfs.fragment_size();
    |                                                                 ^^^^^^^^^^^^^^^^^^^^^^^ expected `u64`, found `u32`
error[E0277]: cannot multiply `u64` by `u32`
   --> src/blocks/disk_space.rs:118:63
    |
118 |         let used = (statvfs.blocks() - statvfs.blocks_free()) * statvfs.fragment_size();
    |                                                               ^ no implementation for `u64 * u32`
    |
    = help: the trait `Mul<u32>` is not implemented for `u64`
    = help: the following other types implement trait `Mul<Rhs>`:
              <&'a u64 as Mul<u64>>
              <&u64 as Mul<&u64>>
              <u64 as Mul<&u64>>
              <u64 as Mul>
error[E0308]: mismatched types
   --> src/blocks/disk_space.rs:119:54
    |
119 |         let available = statvfs.blocks_available() * statvfs.block_size();
    |                                                      ^^^^^^^^^^^^^^^^^^^^ expected `u64`, found `u32`
error[E0277]: cannot multiply `u64` by `u32`
   --> src/blocks/disk_space.rs:119:52
    |
119 |         let available = statvfs.blocks_available() * statvfs.block_size();
    |                                                    ^ no implementation for `u64 * u32`
    |
    = help: the trait `Mul<u32>` is not implemented for `u64`
    = help: the following other types implement trait `Mul<Rhs>`:
              <&'a u64 as Mul<u64>>
              <&u64 as Mul<&u64>>
              <u64 as Mul<&u64>>
              <u64 as Mul>
error[E0308]: mismatched types
   --> src/blocks/disk_space.rs:120:44
    |
120 |         let free = statvfs.blocks_free() * statvfs.block_size();
    |                                            ^^^^^^^^^^^^^^^^^^^^ expected `u64`, found `u32`
error[E0277]: cannot multiply `u64` by `u32`
   --> src/blocks/disk_space.rs:120:42
    |
120 |         let free = statvfs.blocks_free() * statvfs.block_size();
    |                                          ^ no implementation for `u64 * u32`
    |
    = help: the trait `Mul<u32>` is not implemented for `u64`
    = help: the following other types implement trait `Mul<Rhs>`:
              <&'a u64 as Mul<u64>>
              <&u64 as Mul<&u64>>
              <u64 as Mul<&u64>>
              <u64 as Mul>
Some errors have detailed explanations: E0277, E0308.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `i3status-rs` due to 8 previous errors
>>> ERROR: i3status-rust: build failed
```

**Comment by MaxVerevkin** [comment]
Does b2c2bc0 fix it?

**Comment by nekopsykose** [comment]
yep. thank you!

**Comment by MaxVerevkin** [comment]
Released v0.30.1
